mlir/lib/ExecutionEngine/conv-validation-wrappers.cpp

```diff
@@ -558,9 +558,41 @@ extern "C" void mcpuVerifyInt32(int32_t *gpuAllocated, int32_t *gpuAligned,
                                 int64_t gpuStride, int32_t *valAllocated,
                                 int32_t *valAligned, int64_t valOffset,
                                 int64_t valSize, int64_t valStride,
-                                float thr_RMS, float thr_absDiff,
-                                float thr_relDiff, char printDebug) {
+                                char printDebug) {
+  assert(gpuSize == valSize);
+  for (int64_t i = 0; i < valSize; ++i) {
+    if (gpuAligned[i] != valAligned[i]) {
+      if (printDebug)
```

Comment:
Please implement the full range of debug flags here

Reply:
The debug information is quite simple, so I chose to print them all as long as printDebug is on. Do you prefer different debug information for different debug options? What would you suggest?

Reply:
Agree. The debug information for float types don't really make sense for integers.

Reply:
Let me rephrase: we have four debug options: `always`, `failure`, `summary`, and `off`.

All of these options can be meaningfully implemented for the int8 verifier.

`summary` would, for example, be a count of incorrect elements and possibly the largest difference or something like that.

Reply:
We could implement that way, but how useful would those information be? I mean I wouldn't care much about the largest difference or the number of incorrect elements. They may help debugging, but collecting those information in CI sounds wasting time for me.

Reply:
This is important to match CI, which, IIRC, calls for `debug=summary` by default

Reply:
More importantly, when I'm testing, and I set one of those four values for the debug output flag, I expect them to work as specified.

Reply:
OK, if you insist. For 'failure' I'll print out the correct value, incorrect value and difference, and 'summary' is the number of incorrect elements and the largest difference. Anything else?

Reply:
Ok, so, the behaviors are this
- On `always`, or for `failure`, or `summary` when the test isn't successful, print some sort of summary (what you suggested about largest difference  + number of failures is good)
- With `always` or `failure`, print out the individual failing elements.

Reply:
That's what the float verifier does